Learn CSS intro content (#5243)
GoogleChrome/web.dev: src/site/content/en/learn/css/index.md

---
title: Learn CSS
authors:
  - andybell

> description: A course that breaks down the fundamentals of CSS to help you become a well-rounded front-end developer.

Reviewer: ```suggestion
description: An evergreen CSS course and reference to level-up your web styling expertise.
```

Reviewer: What do you all think about this for the description? We use the works "breaks down fundamentals" and "well-rounded front-end developer" in the intro text, so i updated the description text with the course pitch.

date: 2021-05-04
tags:
  - css
  - layout
---

**Welcome to Learn CSS!**

This course breaks down the fundamentals of CSS into digestible,

> easy to understand parts.

Reviewer: ```suggestion
easy to understand pieces.
```

Over the next few modules,
you will learn how the core aspects of CSS work and how to use them effectively in your projects.

You'll learn CSS fundamentals like the box model, cascade and specificity, flexbox, grid and z-index.
And, along with these fundamentals,
you'll learn about functions, color types, gradients, logical properties and inheritance
to make you a well-rounded front-end developer,
ready to take on any user interface.

> Each module is full of interactive demos and self-assessments for you to explore and test your knowledge.

Reviewer: ```suggestion
Each module is full of interactive demos and self-assessments for you to test your knowledge.
```

In addition to learning through reading and demos,

> each module contains an accompanying podcast episode as another way to learn and expand your knowledge.

Reviewer: ```suggestion
there is an accompanying podcast episode for each topic as another way to learn and continue expanding your knowledge.
```

This course is created for beginner and advanced CSS developers alike.

> You can go through the series of modules from start to finish

Reviewer: ```suggestion
You can go through the series from start to finish
```

> to get a well-rounded understanding of CSS from top to bottom,

Reviewer: ```suggestion
to get a broad understanding of CSS from top to bottom,
```

or you can use it as a reference for specific styling subjects.

> For those new to web development, check out the

Reviewer: ```suggestion
For those new to web development overall, check out the
```
